instance/tests/api/test_openedx_appserver.py

```diff
@@ -162,9 +162,10 @@ def test_get_details(self):
         self.assertIn('log_error_entries', data)
 
     @patch_gandi
+    @patch('instance.models.openedx_instance.OpenEdXInstance.provision_rabbitmq')
     @patch('instance.models.openedx_appserver.OpenEdXAppServer.provision', return_value=True)
     @patch('instance.models.mixins.load_balanced.LoadBalancingServer.run_playbook')
-    def test_spawn_appserver(self, mock_run_playbook, mock_provision):
+    def test_spawn_appserver(self, mock_run_playbook, mock_provision, mock_rabbitmq):
```

Comment:
@viadanna It would be worth renaming `mock_rabbitmq` to `mock_provision_rabbitmq` (here and below) so that it's completely clear which method it is mocking.